app/assets/stylesheets/scaffolds.scss

```diff
@@ -12,7 +12,7 @@ pre {
   background-color: #eee;
   padding: 10px;
   font-size: 11px; }
-
+/*
```

Comment:
I thought I would need this code again in the future that is why I commented it out and left it in there but also I did not leave any comments specifying why I left it in there, oopsie. In case I want to keep some code by commenting it out, would adding a comment explaining its purpose would be ok? Thank you!